+nix/MultiTag.m

```diff
@@ -52,17 +52,21 @@
             r = nix.Utils.open_entity(obj, 'openReferences', id_or_name, @nix.DataArray);
         end
 
-        function r = open_reference_idx(obj, idx)
+        function r = open_reference_idx(obj, index)
+            idx = nix.Utils.handle_index(index);
             r = nix.Utils.open_entity(obj, 'openReferenceIdx', idx, @nix.DataArray);
         end
 
-        function r = retrieve_data(obj, pos_idx, id_or_name)
+        function r = retrieve_data(obj, position_idx, id_or_name)
```

Comment:
if we spell out index in all other places, I would vote for doing the same here... other opinions?

Reply:
It was more along the lines of using `idx` in a compound word and `index` if its on its own, but I don't mind either way.

Reply:
fine with me, @achilleas-k  any thoughts?

Reply:
Yeah, I think shortening index to idx when it's coupled with another word is OK, even though we use the full word when it's on its own.